packages/components/src/components/hds/form/masked-input/field.hbs

```diff
@@ -12,16 +11,10 @@
   as |F|
 >
   {{! Notice: the order of the elements is not relevant here, because is controlled at "Hds::Form::Field" component level }}
-  {{yield
-    (hash
-      Label=F.Label
-      isRequired=F.isRequired
-      isOptional=F.isOptional
-      HelperText=F.HelperText
-      Error=F.Error
-      CharacterCount=(component F.CharacterCount value=@value)
-    )
-  }}
+  {{yield (hash Label=F.Label HelperText=F.HelperText Error=F.Error)}}
+  {{#if F.CharacterCount}}
+    {{yield (hash CharacterCount=(component F.CharacterCount value=@value))}}
+  {{/if}}
```

Comment:
This type of safeguard was required by TypeScript to prevent calling a null component when `F.CharacterCount` is `null`.